File: Backend/BankingPaymentsApp_API/BankingPaymentsApp_API/DTOs/ClientUserResponseDTO.cs
namespace BankingPaymentsApp_API.DTOs
{
    public class ClientUserResponseDTO
    {
        public int UserId { get; set; }
        public string UserFullName { get; set; }
        public string UserName { get; set; }
        public int UserRoleId { get; set; }
        public string UserEmail { get; set; }
        public string UserPhone { get; set; }
        public DateTime UserJoiningDate { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Address { get; set; }
        public bool KycVierified { get; set; }
        public int? AccountId { get; set; }
    }
}

File: frontend/BankingPaymentsApp_Frontend/src/app/DTO/ClientUserResponseDTO.ts
export interface ClientUserResponseDTO {
  userId: number;
  userFullName: string;
  userName: string;
  userRoleId: number;
  bankId:number;
  userEmail: string;
  userPhone: string;
  userJoiningDate: string;
  dateOfBirth: string;     
  address: string;
  kycVierified: boolean;
  accountId?: number;
}
